.golangci.goheader.template

```diff
@@ -0,0 +1,13 @@
+Copyright © {{ YEAR }} Meroxa, Inc.
```

Comment:
I _think_ this can be Meroxa, Inc. and GophersLab, but let's check that with Ali.